README.zh-CN.md

```
@@ -0,0 +1,133 @@
+        <img src="assets/mcpInstallv2.png" alt="MCP Configuration">
+        <br>
+        <em>配置 MCP 连接</em>
+      </td>
+    </tr>
+  </table>
+</div>
+
+### 3. 测试连接
+
+在您的编程助手中，发送：
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)